instrumentor/controllers/sourceinstrumentation/namespace_controller.go

```diff
@@ -44,18 +44,21 @@ func (r *NamespaceReconciler) Reconcile(ctx context.Context, req ctrl.Request) (
 		return ctrl.Result{}, err
```

Comment:
6 lines above we have this log message:

```
logger.Info("namespace reconcile - will delete instrumentation for workloads that are not enabled in this namespace", "namespace", req.Name)
```

we should update the message as we are not only deleting now, but can also create. I think the term "sync" is more accurate.